balsam/site/job_source.py

```diff
@@ -134,12 +132,7 @@ def _run(self) -> None:
             logger.debug(f"JobSource queue depth is currently {qsize}. Fetching {fetch_count} more")
             if fetch_count:
                 params = self._get_acquire_parameters(fetch_count)
-                try:
-                    jobs = self.session.acquire_jobs(**params)
-                except (requests.ConnectionError, requests.HTTPError) as exec:
-                    logger.exception("Failed to communicate with server")
-                    logger.exception(str(exec))
-                    continue
+                jobs = self.session.acquire_jobs(**params)
```

Comment:
Perhaps this should still be guarded against a timeout error, because even if the client catches the error, it'll only retry a few times before `backoff()` gives up and re-raises an error, which would crash the site/launcher.

Reply:
Hi Misha! We are still catching the timeout. There is probably another round of site stability that we'll have to do, but this one guards against HTTPErrors which were making sites unresponsive immediately.